From de6016609d4dc4421974562a868b2e53fd06144e Mon Sep 17 00:00:00 2001
From: John Chung <john.chung@mic.com.tw>
Date: Tue, 17 Mar 2020 09:48:03 +0800
Subject: [PATCH] Remove power button and reset button press and long press
 action

Remove power button press and long press bus method call
Remove power button press bus method call
---
 src/button_handler.cpp | 28 ----------------------------
 1 file changed, 28 deletions(-)

diff --git a/src/button_handler.cpp b/src/button_handler.cpp
index 1a26744..385bb4d 100755
--- a/src/button_handler.cpp
+++ b/src/button_handler.cpp
@@ -168,15 +168,6 @@ void Handler::powerPressed(sdbusplus::message::message& msg)
             log<level::ERR>("call IpmiSelAdd failed");
         }
 
-        sdbusplus::message::variant<std::string> state =
-            convertForMessage(transition);
-
-        auto service = getService(HOST_STATE_OBJECT_NAME, hostIface);
-        auto method = bus.new_method_call(
-            service.c_str(), HOST_STATE_OBJECT_NAME, propertyIface, "Set");
-        method.append(hostIface, "RequestedHostTransition", state);
-
-        bus.call(method);
     }
     catch (SdBusError& e)
     {
@@ -217,15 +208,6 @@ void Handler::longPowerPressed(sdbusplus::message::message& msg)
             log<level::ERR>("call IpmiSelAdd failed");
         }
 
-        sdbusplus::message::variant<std::string> state =
-            convertForMessage(Chassis::Transition::Off);
-
-        auto service = getService(CHASSIS_STATE_OBJECT_NAME, chassisIface);
-        auto method = bus.new_method_call(
-            service.c_str(), CHASSIS_STATE_OBJECT_NAME, propertyIface, "Set");
-        method.append(chassisIface, "RequestedPowerTransition", state);
-
-        bus.call(method);
     }
     catch (SdBusError& e)
     {
@@ -246,16 +228,6 @@ void Handler::resetPressed(sdbusplus::message::message& msg)
 
         log<level::INFO>("Handling reset button press");
 
-        sdbusplus::message::variant<std::string> state =
-            convertForMessage(Host::Transition::Reboot);
-
-        auto service = getService(HOST_STATE_OBJECT_NAME, hostIface);
-        auto method = bus.new_method_call(
-            service.c_str(), HOST_STATE_OBJECT_NAME, propertyIface, "Set");
-
-        method.append(hostIface, "RequestedHostTransition", state);
-
-        bus.call(method);
     }
     catch (SdBusError& e)
     {
-- 
2.7.4

